main/client/features/home-chart/home-chart-dir.js

```diff
@@ -0,0 +1,170 @@
+
+    return {
+        restrict: 'EA',
+        templateUrl: './dist/html/home-chart/home-chart-temp.html',
+        controller: 'homeChartCtrl',
+        link: function(scope, elem, attrs) {
+            var dataset = [];
+
+            // Setup data
+            var numDataPoints = 20; // Number of dummy data points
+            var maxRange = Math.random() * 1000; // Max range of new values
+            for (var i = 0; i < numDataPoints; i++) {
+                var newNumber1 = (Math.floor(Math.random() * maxRange) + 25); // New random integer
+                var newNumber2 = (Math.floor(Math.random() * maxRange) + 50); // New random integer
+                dataset.push([newNumber1, newNumber2]); // Add new number to array
+            }
+
+            // Setup settings for graphic
+            var canvas_width = 1000;
+            var canvas_height = 350;
+            var padding = 30; // for chart edges
+
+            console.log("this is dataset", dataset);
+            // Create scale functions
+            var xScale = d3.scale.linear() // xScale is width of graphic
+                .domain([0, d3.max(dataset, d => {
+                    return d[0]; // input domain
+
+                })])
+                .range([padding, canvas_width - padding * 2]); // output range
+
+            var yScale = d3.scale.linear() // yScale is height of graphic
+                .domain([0, d3.max(dataset, function(d) {
+                    return d[1]; // input domain
+                })])
+                .range([canvas_height - padding, padding]); // remember y starts on top going down so we flip
+
+            // Define X axis
+            const xAxis = d3.svg.axis()
+                .scale(xScale)
+                .orient("bottom")
+                .ticks(0);
+
+            // Define Y axis
+            const yAxis = d3.svg.axis()
+                .scale(yScale)
+                .orient("left")
+                .ticks(0);
+
+            // Create SVG element
+            const svg = d3.select("h3") // This is where we put our vis
+                .append("svg")
+                .attr("width", canvas_width)
+                .attr("height", canvas_height)
+
+            //randomize radius size
+            function randomizeRadius () {
+                return Math.round(Math.random() * 25 + 5);
+            }
+			//pick either blue or orange
+            function colorPicker () {
+                var randomizer = Math.floor(Math.random() * 2);
+                console.log("this is randomizer", randomizer);
+                if (randomizer === 0) {
+                    return 'rgba(248, 158, 49, .8)';
+                } else {
+                    return 'rgba(0, 114, 188, .95)'
+                }
+            }
+            // Create Circles
+            svg.selectAll("circle")
+                .data(dataset)
+                .enter()
+                .append("circle") // Add circle svg
+                .attr("cx", function(d) {
```

Comment:
Not mandatory, but try to be consistent in ES6 usage. Try to not swap between fat arrow/standard declaration regularly in the same file unless needed due to scoping.